documentation/src/settings_general.md

```diff
@@ -0,0 +1,8 @@
+# General settings
+In the general settings, you will find settings that influence the behaviour of the program.
+
+{{#include ./settings/capture_delay.md}}
+{{#include ./settings/single_photo_collage.md}}
+
+## Hotkeys
```

Comment:
Although we show the hotkeys in that screen, they are not configurable as of now. Maybe this should be a separate page in the mdbook?